spring-integration-core/src/main/java/org/springframework/integration/store/AbstractKeyValueMessageStore.java

```diff
@@ -44,26 +41,32 @@
 
 	protected static final String MESSAGE_GROUP_KEY_PREFIX = "MESSAGE_GROUP_";
 
+	/**
+	 * Represents the time when the message has been added to the store.
+	 * @deprecated since 5.0. This constant isn't used any more.
+	 */
+	@Deprecated
 	protected static final String CREATED_DATE = "CREATED_DATE";
```

Comment:
Agreed. The same we have with the `MessageDocument` for MongoDB.
That should be enough easy since it is key-value store without restrictions for the target `value`.